### PR TITLE
undo deletion of fields formerly unsupported but now supported

### PR DESCRIPTION
**Pull request by losborne**
Certain fields were removed because they were not yet supported in cql_qdm_patientapi.  Those fields have since been supported, so we can remove the deletion.

Pull requests into cqm-converter require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/BONNIE-1746
- [x] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases N/A
- [x] Tests have been run locally and pass
- [x] If applicable, the library version number in `cqm-converter.gemspec` has been updated N/A

**Bonnie Reviewer:**

Name: @mayerm94
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases NA
- [x] You have tried to break the code

**Cypress Reviewer:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

**Comment by losborne** [comment]
Rubocop failures are not related and are new.  Restarting master branch build revealed same failures.  They should probably be taken care of in a separate PR.

File: lib/cqm/converter/utils.rb [hunk 1]
```diff
@@ -38,19 +38,9 @@ def self.gather_qdm_model_attrs
           qdm_model_attrs[model.to_s].concat(QDM.const_get(model).embedded_relations.keys.map! { |a| a.camelize(:lower) })
         end
       end
-      # TODO: These fields are currently not supported. See:
+      # TODO: This fields is currently not supported. See:
```
Comment (okeefm) [inline lib/cqm/converter/utils.rb]: Technically we should probably singularize `fields`. Otherwise it looks good.

Comment (losborne) [reply lib/cqm/converter/utils.rb]: Doh! Good catch!
